app/models/miq_server/worker_management/monitor/kubernetes.rb

```diff
@@ -0,0 +1,116 @@
+module MiqServer::WorkerManagement::Monitor::Kubernetes
+  extend ActiveSupport::Concern
+  attr_accessor :pod_resource_version
+
+  def current_pods
+    @current_pods ||= Concurrent::Hash.new
+  end
+
+  def reset_current_pods
+    current_pods.clear
```

Comment:
I'm not sure we need a method for this as `current_pods.clear` is as equally clear as `reset_current_pods` if not clearer.

Reply:
Good point.